src/meta-estimator/src/main/scala/MetaEstimator.scala

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.ml.spark
```

Comment:
This looks awfully similar to ``Pipeline`` - could you please look into reusing that if possible?